classes/client/client.php

```diff
@@ -34,9 +34,15 @@ static function create_req($url, $token, $method, $postBody = null) {
         ));
 
         $response = curl_exec($curl);
```

Comment:
@naveenbalachandar - In order to use Guzzle in the plugin, it is required to ask clients to additionally install [Guzzle Moodle plugin](https://github.com/zaeja/moodle-local_guzzle) or include the Guzzle library in the plugin code base.

Considering the complexity of the current HTTP requests, I think it would be better to stick with `curl` for now.